test/__snapshots__/cacheContext-option.test.js.snap

```diff
@@ -22,9 +22,55 @@ Array [
     ],
     "remainingRequest": "test/fixtures/basic/file_1.js",
     "result": Array [
-      "/* eslint-disable */
-console.log('file_1');
-",
+      Object {
+        "data": Array [
+          47,
+          42,
+          32,
+          101,
+          115,
+          108,
+          105,
+          110,
+          116,
+          45,
+          100,
+          105,
+          115,
+          97,
+          98,
+          108,
+          101,
+          32,
+          42,
+          47,
+          10,
+          99,
+          111,
+          110,
+          115,
+          111,
+          108,
+          101,
+          46,
+          108,
+          111,
+          103,
+          40,
+          39,
+          102,
+          105,
+          108,
+          101,
+          95,
+          49,
+          39,
+          41,
+          59,
+          10,
+        ],
+        "type": "Buffer",
```

Comment:
`JSON.parse` won't convert this field to buffer.

https://github.com/webpack-contrib/cache-loader/blob/master/src/index.js#L241

Reply:
Yes, you are right

Reply:
I think we need improve logic with save

Reply:
@malash @evilebottnawi I'm now using `json-buffer` to parse and stringify our objects which solves the problem with Buffers and also encodes them with base64 on stringify.